### PR TITLE
add EC transparency in decision making

### PR DESCRIPTION
**Pull request by orchid00**
this PR closes this issue https://github.com/carpentries/executive-council-info/issues/61 which has been approved by the EC
Suggest to be reviewed by the Core Team for readability

### CODE REVIEW

**Comment by maneesha** [comment]
This looks good to me.  @orchid00  do you want to incorporate the suggestion from @lexnederbragt ?

**Review by kmomar — APPROVED** [review]
Looks good to me as well with that suggested change.

File: topic_folders/governance/executive-council.md [hunk 1]
```diff
@@ -121,6 +121,31 @@ liaisons to Committees or Task Forces may have additional commitments for meetin
 agenda items for meetings, responding to email requests for opinions, and formal voting on resolutions (as issues in this
 repository), and work resulting from being on a task force or committee representing the Executive Council.
 
+#### Executive Council transparency in decision making
+
+The Executive Council, the Core Team and the Executive Director agreed on the following points:
+
```
[the remark this answers is not in the document]
Comment (orchid00) [reply topic_folders/governance/executive-council.md:127]: it is linked

**Comment by orchid00** [comment]
Thanks @maneesha @lexnederbragt @kmomar 
https://github.com/carpentries/executive-council-info/issues/61 is the issue to be closed with this PR
it is linked already.

File: topic_folders/governance/executive-council.md [hunk 2]
```diff
@@ -121,6 +121,31 @@ liaisons to Committees or Task Forces may have additional commitments for meetin
 agenda items for meetings, responding to email requests for opinions, and formal voting on resolutions (as issues in this
 repository), and work resulting from being on a task force or committee representing the Executive Council.
 
+#### Executive Council transparency in decision making
+
+The Executive Council, the Core Team and the Executive Director agreed on the following points:
```
Comment (lexnederbragt) [inline topic_folders/governance/executive-council.md:126]:
```suggestion
The Executive Council, the Core Team and the Executive Director [agreed](https://github.com/carpentries/executive-council-info/issues/61) on the following points:
```

File: topic_folders/governance/executive-council.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lexnederbragt) [reply topic_folders/governance/executive-council.md:127]: I meant adding a hyperlink, as in the suggestion I just added.

**Comment by maneesha** [comment]
I think this is good to merge now.

**Pull request merged**